README.md

```diff
@@ -397,14 +398,124 @@ constructor(props) {
 
 # Lists
 
-> TODO
-> ng-repeat, trackby vs key
+### AngularJS
+```js
+export class BookListComponentCtrl {
+  constructor() {
+    this.books = [
+      {
+        id: 1,
+        title: "Eloquent JavaScript",
+        author: "Marijn Haverbeke"
+      },
+      {
+        id: 2,
+        title: "JavaScript: The Good Parts",
+        author: "Douglas Crockford"
+      },
+      {
+        id: 3,
+        title: "JavaScript: The Definitive Guide",
+        author: "David Flanagan"
+      }
+    ];
+  }
+}
+```
+
+```html
+<ul>
+  <li ng-repeat="book in $ctrl.books track by book.id">
+    {{ book.title }} by {{ book.author }}
+  </li>
+</ul>
+```
+
+### Angular
+```js
+@Component({
+  selector: 'book-list',
+  template: `
+    <ul>
+      <li *ngFor="let book of books; trackBy: trackById">
+        {{ book.title }} by {{ book.author }}
+      </li>
+    </ul>
+  `
+})
+export class BookListComponent {
+  this.books = [
+    {
+      id: 1,
+      title: "Eloquent JavaScript",
+      author: "Marijn Haverbeke"
+    },
+    {
+      id: 2,
+      title: "JavaScript: The Good Parts",
+      author: "Douglas Crockford"
+    },
+    {
+      id: 3,
+      title: "JavaScript: The Definitive Guide",
+      author: "David Flanagan"
+    }
+  ];
+
+  trackById(book) {
+    return book.id;
+  }
+}
+```
+
+### React
+```js
+class BookList extends React.Component {
+  constructor() {
+    this.state = {
+      books: [
+        {
+          id: 1,
+          title: "Eloquent JavaScript",
+          author: "Marijn Haverbeke"
+        },
+        {
+          id: 2,
+          title: "JavaScript: The Good Parts",
+          author: "Douglas Crockford"
+        },
+        {
+          id: 3,
+          title: "JavaScript: The Definitive Guide",
+          author: "David Flanagan"
+        }
+      ]
+    };
+  }
+
+  render() {
+    const books = this.state.books;
+
+    return (
+      <ul>
+        books.map(book => {
+          return (
+            <li key="id">
```

Comment:
Should be `key={ book .id }`

Reply:
Fixed, sorry, most of time using ng4 in the last project, React no so often.